frontend/.babelrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     },
     plugins: [
         "relay",
+        "@babel/plugin-proposal-nullish-coalescing-operator",
```

Comment:
By the way, **if** we really need this, it should be a dependency as well.

Reply:
I did some digging and this might actually be necessary right now. The plugin in question doesn't actually run anymore in our configuration. Maybe this is because of the dynamic nature of `preset-env` :shrug:

And then Webpack seems to choke on the syntax, which I don't quite understand, yet. Node knows about it by now, but maybe Webpack uses some old parser library. :shrug: